src/components/PlioQuestion.vue

```diff
@@ -247,14 +265,16 @@ export default {
         this.checkAnswer();
         this.showResult();
       }, loadTime);
+
+      this.$emit("answer-submitted")
```

Comment:
Earlier, nothing was being emitted when `submit` was clicked. The response was being emitted when `close` was clicked. Why change it now?
Because we want to know if the user has clicked submit -> and then mark that part of tutorial complete.
similarly, we want to know if the user has clicked close   -> and then mark that part of tutorial complete

Reply:
Cool 👍🏾